kuma/users/newsletter/checks.py

```diff
@@ -29,8 +29,62 @@ def sendinblue_check(app_configs, **kwargs):
             return [
                 Error(
                     f"Error when creating sendinblue attribute: {response.json()['message']}",
-                    id=SENDINBLUE_ERROR,
+                    id=SENDINBLUE_API_ERROR,
                 )
             ]
 
     return []
+
+
+# We're using sendinblue for marketing emails. Another possible use-case is "transactional"
+# But as we're not using it that way, we don't have to check for that.
+EMAIL_TYPE = "marketing"
+
+
+def create_sendinblue_unsubscribe_webhook():
+    url_path = reverse("api.v1.sendinblue_hooks")
+    url = (
+        "https://" + settings.CUSTOM_WEBHOOK_HOSTNAME + url_path
+        if settings.CUSTOM_WEBHOOK_HOSTNAME
+        else absolutify(url_path)
+    )
+
+    # From https://developers.sendinblue.com/reference#createwebhook
+    # "Occurs whenever a user unsubscribes through an email's subscription management link."
+    events = ("unsubscribed",)
+
+    response = sendinblue.request("GET", "webhooks", params={"type": EMAIL_TYPE})
+
+    no_webhooks_found = response.status_code == 404
+    if not response.ok and not no_webhooks_found:
+        return [
+            Error(
+                f"Error getting sendinblue webhooks: {response.status_code}",
+                id=SENDINBLUE_API_ERROR,
+            )
+        ]
+
+    if not no_webhooks_found:
+        for webhook in response.json()["webhooks"]:
+            if webhook["url"] == url and set(events) == set(webhook["events"]):
```

Comment:
Shouldn't that be:
```suggestion
            if webhook["url"] == url and set(events) & set(webhook["events"]):
```
I.e. an overlap.

Reply:
my bad reason for doing it: I did the same thing we do when setting up the stripe webhook. But I agree that a subset of the events would be enough. Kinda. Then again, we would then also tolerate webhooks that send more events than we can't handle, so maybe equality checking is better in that regard?

Reply:
I'm gonna merge the PR because I'm not leaning either way on this one but I feel like it's a discussion worth having (or keeping in mind)